policyengine_canada/variables/gov/provinces/bc/tax/income/credits/bc_age_credit/bc_age_credit.py

```diff
@@ -0,0 +1,20 @@
+from policyengine_canada.model_api import *
+
+
+class bc_age_credit(Variable):
+    value_type = float
+    entity = Person
+    label = "British Columbia Age credit, non-refundable"
+    unit = CAD
+    definition_period = YEAR
```

Comment:
Alternatively, split this into `bc_age_credit_eligible` and use that in `defined_for`
```suggestion
    definition_period = YEAR
    defined_for = ProvinceCode.BC
```